File: backend/compute.php
<?php
$n = intval($argv[1] ?? 20);
$count=0; $num=1; $sum=0;
while ($count < $n) { $num++; $p=true; for ($i=2;$i*$i<=$num;$i++){ if ($num%$i==0){$p=false;break;} } if ($p){ $sum+=$num; $count++; } }
echo json_encode(["lang"=>"PHP","what"=>"sum of first $n primes","result"=>$sum]);

File: backend/Compute.java
public class Compute {
  public static void main(String[] a) {
    int n = a.length > 0 ? Integer.parseInt(a[0]) : 20;
    long x = 0, y = 1;
    for (int i = 0; i < n; i++) { long t = x + y; x = y; y = t; }
    System.out.println("{\"lang\":\"Java\",\"what\":\"fibonacci(" + n + ")\",\"result\":" + x + "}");
  }
}
